examples/CMakeLists.txt

```diff
@@ -17,8 +17,9 @@ set(ENV{PKG_CONFIG_PATH} "${PKG_CONFIG_CHANGED_PATH}")
 
 find_package(LibtorrentRasterbar REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
+find_package(OpenSSL REQUIRED)
```

Comment:
OPENSSL_INCLUDE_DIR is set by the `find_package(OpenSSL)` call. If that's specified without REQUIRED and OpenSSL isn't found, OPENSSL_INCLUDE_DIR will be an empty string and not cause problems.